common/src/stack/command/stack/argument_processors/pallet.py

```diff
@@ -160,12 +172,17 @@ def run_pallet_hooks(self, operation, pallet_info):
 				Log(f'Env:\n{environ}')
 				Log(f'Stdout:\n{result.stdout}')
 				Log(f'Stderr:\n{result.stderr}')
+				logger.info('Result of running %s (rc=%s):', hook, result.returncode)
```

Comment:
I just noticed the logging in this file is sort of silly.  It currently logs to /var/log/local0, which is fine but I still think it's valuable to have the pallet hook logging in one place.

I think a general cleanup of the logging here can be punted for the time being, but to keep the repetition down, I'd be inclined to make a short wrapper function for this module which takes a string and calls `Log()` and `logger.info()` with it.  Possibly even `self.notify()`.  What do you think?

Maybe something like:

```
def _log_route(msg, log_methods):
    [_logger(msg) for _logger in log_methods]

_log_route(f'stderr: {result.stderr}', [logger.info, Log])
_log_route(some_other_error, [logger.warning, self.notify])
```